CODEOWNERS

```diff
@@ -373,6 +373,7 @@ Makefile* @cilium/build
 /netlify.toml @cilium/ci-structure
 /operator/ @cilium/operator
 /operator/pkg/ciliumenvoyconfig @cilium/sig-servicemesh
+/operator/pkg/controller-runtime @cilium/sig-servicemesh
```

Comment:
I think this commit can be squashed with the commit introducing the pkg

Reply:
agree -> squashed